### PR TITLE
added the avatar component

### PR DESCRIPTION
**Pull request by Rohitprasad83**
avatar

### CODE REVIEW

File: avatar/alert.css [hunk 1]
```diff
@@ -0,0 +1,129 @@
+@import url(../styles.css);
+
+html {
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    background-color: var(--bg-color);
```
Comment (anupkgurung) [inline avatar/alert.css]: instead of bg-color color name would be an appropriate option eg-(--bg-green)

**Review by anupkgurung — COMMENTED** [review]
using px might get you in trouble when dealing responsiveness, that is what i think rest all good.

**Review by ranitasahaa1230 — APPROVED** [review]
Looks good to me. You can do some small changes if required.

File: avatar/alert.css (hunk 1 above)
Comment (Rohitprasad83) [reply avatar/alert.css]: I have made the changes, could you please review them

**Comment by Naveenchandar** [comment]
Looks good.

**Review by maitrakhatri — APPROVED** [review]
looks great !!

**Review by anupkgurung — APPROVED** [review]
seems good.

**Comment by arpit01923** [comment]
looks great

**Pull request merged**